src/components/settings/Settings.cpp

```diff
@@ -7,6 +11,11 @@ using namespace Pinetime::Controllers;
 Settings::Settings(Pinetime::Controllers::FS& fs) : fs {fs} {
 }
 
+void CleanUpQcbor(QCBORDecodeContext* decodeCtx) {
```

Comment:
Maybe you wish to split it off to some Util class so that both Weather and Settings can use it?

Reply:
That's probably a good idea, but I have no idea how to add a new class to the code and use it from both places. As I mentioned, I do not know C++ very well.